samples/hello-world/HW-1qbit.ipynb

```diff
@@ -0,0 +1,260 @@
+        "nteract": {
+          "transient": {
+            "deleting": false
+          }
+        }
+      },
+      "outputs": [],
+      "source": [
+        "from azure.quantum import Workspace\n",
+        "\n",
+        "workspace = Workspace (\n",
+        "    subscription_id = \"\",\n",
+        "    resource_group = \"\",\n",
+        "    name = \"\",\n",
+        "    location = \"\"\n",
+        ")"
+      ]
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "nteract": {
+          "transient": {
+            "deleting": false
+          }
+        }
+      },
+      "source": [
+        "Let's use the `Workspace` object to view the available _targets_."
+      ]
+    },
+    {
+      "cell_type": "code",
+      "execution_count": null,
+      "metadata": {},
+      "outputs": [],
+      "source": [
+        "targets = workspace.get_targets()\n",
+        "print(\"This workspace's targets:\")\n",
+        "for target in targets:\n",
+        "    print(\"-\", target.name)\n"
+      ]
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "nteract": {
+          "transient": {
+            "deleting": false
+          }
+        }
+      },
+      "source": [
+        "The resulting list should contain a list of target names, along with some details for each one. A _target_ is a remote solver that can perform optimization or quantum computation. In this example, we will focus on optimization."
+      ]
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "nteract": {
+          "transient": {
+            "deleting": false
+          }
+        }
+      },
+      "source": [
+        "### ❕ Do you see `1qbit.tabu` in your list of targets? If so, you're ready to keep going.\n",
+        "\n",
+        "Don't see it? You may need to add 1QBit to your workspace to run this sample. Navigate to the **Providers** page in the portal and click **+Add** to add the 1QBit provider."
+      ]
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "nteract": {
+          "transient": {
+            "deleting": false
+          }
+        }
+      },
+      "source": [
+        "### 1QBit: The optimization provider\n",
+        "Azure Quantum connects with first- and third-party optimizers to deliver optimization solutions. These offerings are called *providers*. Each provider can offer multiple *targets* with different capabilities. See the table below for 1QBit's targets.\n",
+        "\n",
+        "Optimization Algorithm | Target ID | Python Class\n",
+        "| --- | ---| ---| --- | -- |\n",
+        "[Tabu search](https://docs.microsoft.com/azure/quantum/provider-1qbit#tabu-search-solver) | `1qbit.tabu` |  `azure.quantum.optimization.oneqbit.TabuSearch` |\n",
+        "[Path relinking](https://docs.microsoft.com/azure/quantum/optimization-parallel-tempering) | `1qbit.pticm`  | `azure.quantum.optimization.oneqbit.PathRelinkingSolver`\n",
```

Comment:
Path relinking URL points to our solver - should be: "https://docs.microsoft.com/en-us/azure/quantum/provider-1qbit#path-relinking-solver" ?